sys-apps/firejail/firejail-0.9.66.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+
+inherit toolchain-funcs python-single-r1 linux-info
+
+if [[ ${PV} != 9999 ]]; then
+	SRC_URI="https://github.com/netblue30/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/netblue30/firejail.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="Security sandbox for any type of processes"
+HOMEPAGE="https://firejail.wordpress.com/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="apparmor +chroot contrib +dbusproxy +file-transfer +globalcfg +network +private-home +suid test +userns +whitelist X"
+# Needs a lot of work to function within sandbox/portage
+# bug #769731
+RESTRICT="test"
+
+RDEPEND="!sys-apps/firejail-lts
+	apparmor? ( sys-libs/libapparmor )
+	contrib? ( ${PYTHON_DEPS} )
+	dbusproxy? ( sys-apps/xdg-dbus-proxy )"
+
+DEPEND="${RDEPEND}
+	sys-libs/libseccomp
+	test? ( dev-tcltk/expect )"
+
+REQUIRED_USE="contrib? ( ${PYTHON_REQUIRED_USE} )"
+
+pkg_setup() {
+	CONFIG_CHECK="~SQUASHFS"
+	local ERROR_SQUASHFS="CONFIG_SQUASHFS: required for firejail --appimage mode"
+	check_extra_config
+	use contrib && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	find -type f -name Makefile.in -exec sed -i -r -e '/CFLAGS/s: (-O2|-ggdb) : :g' {} + || die
+
+	sed -i -r -e '/CFLAGS/s: (-O2|-ggdb) : :g' ./src/common.mk.in || die
+
+	# fix up paths to templates and docs
+	grep -l -Z -r -e '/usr/share/doc/firejail[^-]' ./RELNOTES ./src/man/ ./etc/profile*/ ./test/ | xargs -0 sed -i -r -e "s:/usr/share/doc/firejail([^-]|\$):/usr/share/doc/${PF}\1:"
```

Comment:
It's a long line so in cases like this, I'd usually prefer to shove the grep output into a bash array, then iterate over it (or just call sed with the argument).

It's easier to read if we do that and it also makes it easier to add missing || dies, which we need when we call external commands to ensure the ebuild stops in the event of failure.

So, something like:
```
local files=$( grep -l ... || die)
local file
# fix up ...
for file in ${files[@]} ; do
	sed ... || die
done
```

Or you run that huge thing locally and turn it into a patch (git init && git commit -m "initial" && ... && git diff ...).

Reply:
Good call; I went with the local array & loop approach. I want to make something that doesn't have to be adjusted each time a new mention of the templates is added to some particular profile file, etc.

Reply:
> because if it gets no hits I'm not sure that's a bad thing

Eh, I take that back; if the grep gets no hits in a future firejail version, that means either a) it's failing for some subtle reason that I should look into and fix, or b) it's no longer needed so I should look into and remove. So, updating again...

Reply:
You beat me to it! :)